drools-game-engine-horserace-kjar/src/main/resources/rules/game/gamerules.drl

```diff
@@ -0,0 +1,76 @@
+package org.drools.game.horserace;
+import org.drools.game.core.api.*;
+import org.drools.game.model.api.*;
+import org.drools.game.horserace.model.*;
+
+global GameMessageService messageService;
+global GameCallbackService callback;
+
+rule "Greeter"
+    when
+        $player : Player()
+    then
+        insert(  messageService.newGameMessage(  "Welcome, " + $player.getName() + ". Venture forth to the stables and mount a mighty steed!" ) );
+end
+
+rule "Give the player a team"
```

Comment:
This rule is defining that you will create a player per team, is that correct? If you have a team per player, you can just use the player itself, right? Are you planning to add more players to that team? Then you need to improve that rule